src/components/overlays/SavePrompt.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import _ from 'lodash';
+import React from 'react';
+import { FiArrowRight } from 'react-icons/fi';
+import { getLabel } from '~/codeLists/labelCodes';
+import * as s from './savePrompt.scss'; // should use savePrompt.scss
+
+const SavePrompt = (props: any) => {
+    const changesContent = () => {
```

Comment:
Our convention has been "if some method returns JSX, it should start with `render` prefix"